sdk/core/azure-core-http-jdk-httpclient/src/main/java/com/azure/core/http/jdk/httpclient/JdkAsyncHttpClient.java

```diff
@@ -160,7 +160,7 @@ private int getJavaVersion() {
             }
             try {
                 return Integer.parseInt(version.substring(2, 3));
-            } catch (Throwable t) {
+            } catch (Exception t) {
                 throw LOGGER.logExceptionAsError(new RuntimeException("Can't parse 'java.version':" + version, t));
```

Comment:
- we should rename `t` to `e`
- another improvement. If `t` is already `RuntimeException` we don't have to wrap it. Perhaps we need another method in the logger and encapsulate it there.